docs/iot/iot-partners/anchor.md

```diff
@@ -0,0 +1,43 @@
+# Tizen IoT with Anchor SoM Module
```

Comment:
**Note:**
Please update the toc_all.md file. This page is a new page, so it need to be added into the TOC for IoT. This page should be be placed at ‘IoT-partners > Anchor’

Reply:
updated and added into toc_all.md